### ISSUE TITLE
Can't compile, missing header

### ISSUE THREAD

**Issue by j0hn** [wimleers/sudoku#2]
I've got the following error while trying to complie it under linux (Ubuntu 10.04 and g++ 4.4.3)

```
$ make
g++ -c -pipe -O2 -Wall -W -D_REENTRANT -DQT_NO_DEBUG -DQT_GUI_LIB -DQT_CORE_LIB -DQT_SHARED -I/usr/share/qt4/mkspecs/linux-g++ -I. -I/usr/include/qt4/QtCore -I/usr/include/qt4/QtGui -I/usr/include/qt4 -I.moc -o .obj/Board.o Board.cpp
Board.cpp: In member function ‘bool Board::Import(const std::string&)’:
Board.cpp:137: error: ‘atoi’ was not declared in this scope
make: *** [.obj/Board.o] Error 1
$
```

i fixed it adding:

```
#include <cstdlib>
```

to Board.cpp

**Comment by wimleers** [comment]
Interesting. This must be a linux-specific include then. I'm going to wrap the include in a Q_OS_LINUX #ifdef so it's only included on Linux, but otherwise, thanks for the report!

**Comment by wimleers** [comment]
Include cstdlib on Linux. Closed by 2397bf50dac06e136eb180275844ea9a0e1d4266.
